app/src/main/java/com/github/se/polyfit/ui/screen/OverviewScreen.kt

```diff
@@ -17,173 +17,133 @@ import androidx.activity.result.ActivityResult
+              modifier = Modifier.align(Alignment.Bottom))
         }
-
-    Button(
-        onClick = Button2,
-        border = BorderStroke(2.dp, MaterialTheme.colorScheme.primaryContainer),
-        elevation = ButtonDefaults.buttonElevation(defaultElevation = 4.dp),
-        colors =
-            ButtonDefaults.buttonColors(
-                contentColor = MaterialTheme.colorScheme.primary,
-                containerColor = MaterialTheme.colorScheme.primaryContainer),
-        modifier =
-            Modifier.align(Alignment.BottomStart)
-                .padding(start = 130.dp, bottom = 15.dp)
-                .background(Color.Transparent)
-                .testTag("EditButton")) {
-          Icon(
-              painter = painterResource(R.drawable.baseline_mode_edit_outline),
-              contentDescription = "penIcon")
-        }
-
-    Button(
-        onClick = Button3,
-        elevation = ButtonDefaults.buttonElevation(defaultElevation = 4.dp),
-        border = BorderStroke(2.dp, MaterialTheme.colorScheme.primaryContainer),
-        colors =
-            ButtonDefaults.buttonColors(
-                contentColor = MaterialTheme.colorScheme.primary,
-                containerColor = MaterialTheme.colorScheme.primaryContainer),
-        modifier =
-            Modifier.align(Alignment.BottomStart)
-                .padding(start = 235.dp, bottom = 15.dp)
-                .background(Color.Transparent)
-                .testTag("HistoryButton")) {
-          Icon(
-              painter = painterResource(R.drawable.outline_assignment),
-              contentDescription = "historyIcon")
+        Spacer(modifier = Modifier.width(5.dp)) // Fixed line
+        Column(modifier = Modifier.align(Alignment.CenterVertically)) {
+          meals.forEach { (meal, calories) ->
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween) {
+                  Text(
+                      meal.toCapitalizedString(),
+                      fontSize = 12.sp,
+                      color = MaterialTheme.colorScheme.secondary)
+                  Text(
+                      "${calories.toInt()}",
+                      fontSize = 12.sp,
+                      color = MaterialTheme.colorScheme.secondary)
+                }
+          }
         }
+      }
+      Text(
+          text = "Tack your meals",
+          fontSize = 25.sp,
+          fontWeight = FontWeight.Bold,
+          color = MaterialTheme.colorScheme.secondary)
+      Spacer(modifier = Modifier.height(16.dp))
+      Row(
+          modifier = Modifier.fillMaxWidth().padding(vertical = 8.dp),
+          horizontalArrangement = Arrangement.SpaceAround) {
+            GradientButton(
+                onClick = onPhoto,
+                active = true,
+                icon = {
+                  Icon(
+                      painterResource(R.drawable.outline_photo_camera),
```

Comment:
same maybe set the contentDescription, for the test and I think for disable person api that help them to navigate through apps the api read contentdescription, so if we want to add these feature later for more inclusivity it can be a good idea